### PR TITLE
November 2023 Release Notes

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
Release notes for November 2023

### CODE REVIEW

**Comment by nataliekwong** [comment]
@katmarkham I mentioned a few highlights in the notes that I thought were meaningful from looking at the PRs that were closed in November, but could you double check this looks good to you too?

File: docs/release_notes/november_2023.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- **Continuous heartbeat checks** We're continually monitoring syncs to ensure they continue making progress, and have added functionality in the background to ensure that we continue receiving updated ["heartbeat" messages](/understanding-airbyte/heartbeats.md) from our connectors. This will ensure that we continue delivering data and avoid any timeouts. 
+
+## Connector Improvements
+
+In addition to being able to extract text content from unstructured data sources, we have also:
+
+ - Revamped core Marketing connectors Pinterest, Instagram and Klaviyo to significantly improve the experience in setting up and ensure resiliency and reliability.
+ - [Added incremenetal sync](https://github.com/airbytehq/airbyte/pull/32473) functionality for Hubspot's stream `property_history`, which improves syncing time and reliability.
```
Comment (katmarkham) [inline docs/release_notes/november_2023.md:21]: sync time and reliability

File: docs/release_notes/november_2023.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Platform Releases
+- **SSO and RBAC** You can now use SSO in Airbyte Cloud to administer permissions in Airbyte. This is currently only available through Okta, with plans to support Active Directory next. We also now offer **RBAC** (admin roles only) to ensure a high level of security when managing you workspace. For access to this feature, reach out to our [Sales team](https://www.airbyte.com/company/talk-to-sales).
+- **Continuous heartbeat checks** We're continually monitoring syncs to ensure they continue making progress, and have added functionality in the background to ensure that we continue receiving updated ["heartbeat" messages](/understanding-airbyte/heartbeats.md) from our connectors. This will ensure that we continue delivering data and avoid any timeouts. 
+
+## Connector Improvements
+
+In addition to being able to extract text content from unstructured data sources, we have also:
+
+ - Revamped core Marketing connectors Pinterest, Instagram and Klaviyo to significantly improve the experience in setting up and ensure resiliency and reliability.
```
Comment (katmarkham) [inline docs/release_notes/november_2023.md:20]: - Revamped core Marketing connectors Klaviyo, Pinterest, and Instagram to significantly improve the setup experience and ensure resiliency and reliability.

**Review by katmarkham — DISMISSED** [review]

File: docs/release_notes/november_2023.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# November 2023
+## airbyte v0.50.34 to v0.50.35
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## ✨ Highlights
+
+Airbyte now supports extracting text content from PDF, Docx, and Pptx files from S3, Azure Blob Storage, and the newly introduced [Google Drive](/integrations/sources/google-drive.md) source. This is an important part of supporting LLM use use cases that rely on unstructured data in files.
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/release_notes/november_2023.md:8]: Good catch, thank you!

File: docs/release_notes/november_2023.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+SSO and RBAC (admin roles only) are now available in Airbyte Cloud! Read more below.
+
+## Platform Releases
+- **SSO and RBAC** You can now use SSO in Airbyte Cloud to administer permissions in Airbyte. This is currently only available through Okta, with plans to support Active Directory next. We also now offer **RBAC** (admin roles only) to ensure a high level of security when managing you workspace. For access to this feature, reach out to our [Sales team](https://www.airbyte.com/company/talk-to-sales).
+- **Continuous heartbeat checks** We're continually monitoring syncs to ensure they continue making progress, and have added functionality in the background to ensure that we continue receiving updated ["heartbeat" messages](/understanding-airbyte/heartbeats.md) from our connectors. This will ensure that we continue delivering data and avoid any timeouts.
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/release_notes/november_2023.md:14]: Adjusted

File: docs/release_notes/november_2023.md (hunk 2 above)
Comment (nataliekwong) [reply docs/release_notes/november_2023.md:20]: +1

File: docs/release_notes/november_2023.md (hunk 1 above)
Comment (nataliekwong) [reply docs/release_notes/november_2023.md:21]: +1

**Review by timroes — APPROVED** [review]

**Pull request merged**